docker/android-base/Dockerfile

```diff
@@ -1,59 +1,20 @@
-FROM ubuntu:14.04
+FROM java:openjdk-6-jdk
```

Comment:
Have you tried running the android-studio after making this change to android-base? I remember Android Studio yelled at me for trying to use openjdk6.